functions/snippets/src/test/java/HelloBackgroundSampleTest.java

```diff
@@ -14,43 +14,26 @@
  * limitations under the License.
  */
 
-import static org.hamcrest.CoreMatchers.containsString;
-import static com.google.common.truth.Truth.assertThat;
-import static org.mockito.Mockito.*;
-import static org.mockito.Mockito.times;
-
-import com.google.gson.Gson;
-import java.io.BufferedReader;
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.PrintStream;
-import java.io.BufferedWriter;
-import java.io.StringReader;
-import java.io.StringWriter;
-import java.nio.charset.Charset;
-import java.util.Base64;
-import java.util.Optional;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-import javax.servlet.ServletInputStream;
 import com.google.cloud.functions.HttpRequest;
 import com.google.cloud.functions.HttpResponse;
-
-import org.junit.contrib.java.lang.system.EnvironmentVariables;
-
-import com.google.common.truth.Truth;
+import com.google.gson.Gson;
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Rule;
 import org.junit.Test;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
 import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.powermock.api.mockito.PowerMockito;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
+import java.io.*;
```

Comment:
I think we should be using Google Style for these? Which means no wildcard imports, no blank line between groups of packages, and static imports before non-static ones (with a blank line separating).

Reply:
+1 - the checkstyles package should be enforcing this?